src/frontend/components/PubPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { kernel, useFriendsList } from '../db'
+import VibeList from './Vibe.jsx'
+
+export default function Pubs () {
+  const peers = useFriendsList()
+  const icons = {
+    0: '♀️',
+    1: '♂️',
+    2: '⚧️'
+  }
+  const sendVibe = (peer) => {
+    console.log('pk0',peer.pk)
+    kernel.sendVibe(peer.pk)
+      .then(chatId => {
+        console.log(chatId)
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+  return (
+    <>
+      <div className='hero'>
+        <h1 className='bar'>Pub name is <strong>"HardRock"</strong></h1>
+        <h3 className='membersList'>Here is memmbers list which is at this bar now</h3>
+        <VibeList />
+        <div className='friendsList friendsListWrap'>
+          {peers.map(friend => (
```

Comment:
refactor for consistency:
rename `friend` => `peer`

`useFriendsList()` will be renamed in future to `usePeers()`